js/libs/ui-shared/src/user-profile/LocaleSelector.tsx

```diff
@@ -20,9 +20,10 @@ export const LocaleSelector = ({
   supportedLocales,
 }: LocaleSelectorProps) => {
   const locales = supportedLocales.map((locale) => ({
-    key: locale,
-    value: localeToDisplayName(locale) || "",
-  }));
+      key: locale,
+      value: t("locale_" + locale, localeToDisplayName(locale) || locale),
```

Comment:
We can use some more modern JS syntax here.
```suggestion
      value: t(`locale_ ${locale}`, localeToDisplayName(locale) ?? locale),
```